tests/unit_test/twitter_ads_api/twitter_ads_api_config_test.py

```diff
@@ -2,66 +2,152 @@
+        config = TwitterAdsApiConfig.parse_config_list_from_dict(CONFIG_DICT)
+        assert config[0].source.resource == RESOURCE_1
+        assert config[1].source.resource == RESOURCE_2
 
     def test_should_read_target_project_dataset_and_table_name(self):
-        config = TwitterAdsApiConfig.from_dict(CONFIG_DICT)
-        assert config.target.project_name == PROJECT_NAME
-        assert config.target.dataset_name == DATASET_NAME
-        assert config.target.table_name == TABLE_NAME
+        config = TwitterAdsApiConfig.parse_config_list_from_dict(CONFIG_DICT)
+        assert config[0].target.project_name == PROJECT_NAME
+        assert config[0].target.dataset_name == DATASET_NAME
+        assert config[0].target.table_name == TABLE_NAME
 
     def test_should_read_api_secrets(self):
         secrets = {'key1': 'value1', 'key2': 'value2'}
-        config = TwitterAdsApiConfig.from_dict(get_config_for_item_config_dict({
-            **ITEM_CONFIG_DICT,
-            'source': {
-                **SOURCE_CONFIG,
-                'secrets': secrets
-            }
-        }))
-        assert config.source.secrets.mapping == secrets
+        config = TwitterAdsApiConfig.parse_config_list_from_dict(
+            get_config_for_item_config_dict(
+                {
+                    **ITEM_CONFIG_DICT_1,
+                    'source': {
+                        **SOURCE_CONFIG_1,
+                        'secrets': secrets
+                    }
+                },
+                {
+                    **ITEM_CONFIG_DICT_2,
+                    'source': {
+                        **SOURCE_CONFIG_2,
+                        'secrets': secrets
+                    }
+                }
+            )
+        )
+        assert config[0].source.secrets.mapping == secrets
+        assert config[1].source.secrets.mapping == secrets
 
     def test_should_read_api_secrets_by_key(self):
         secrets = {'key1': 'value1', 'key2': 'value2'}
-        config = TwitterAdsApiConfig.from_dict(get_config_for_item_config_dict({
-            **ITEM_CONFIG_DICT,
-            'source': {
-                **SOURCE_CONFIG,
-                'secrets': secrets
-            }
-        }))
-        assert config.source.secrets.mapping['key1'] == secrets['key1']
+        config = TwitterAdsApiConfig.parse_config_list_from_dict(
+            get_config_for_item_config_dict(
+                {
+                    **ITEM_CONFIG_DICT_1,
+                    'source': {
+                        **SOURCE_CONFIG_1,
+                        'secrets': secrets
+                    }
+                },
+                {
+                    **ITEM_CONFIG_DICT_2,
+                    'source': {
+                        **SOURCE_CONFIG_2,
+                        'secrets': secrets
+                    }
+                }
+            )
+        )
+        assert config[0].source.secrets.mapping['key1'] == secrets['key1']
```

Comment:
this test looks identical to the test above, except the assert on a specific key?